spaces/domains.json

```
@@ -1,4 +1,4 @@
-{
+{ "voting.cafeswap.finance":"cafeswapfinance.eth",
```

Please correct the indentation in the file, thank you.
```suggestion
{ 
  "voting.cafeswap.finance": "cafeswapfinance.eth",
```

---

I have corrected the indentation issue, hope it has been fixed.
Thank You !